### PR TITLE
FB-16 Added .github

### PR DESCRIPTION
**Pull request by xscotophilic**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/PRIMARY.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] Automated Testing
+
+**Test Configuration**:
+
+* SDK:
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
```
Comment (ngo100ping) [inline .github/PULL_REQUEST_TEMPLATE/PRIMARY.md]: Should be 
- [ ] I have performed a self-review of my code

File: .github/PULL_REQUEST_TEMPLATE/PRIMARY.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (ngo100ping) [inline .github/PULL_REQUEST_TEMPLATE/PRIMARY.md]: Should be
- [ ] I have commented on my code, particularly in hard-to-understand areas

File: .github/SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+We take the security of our software products and services seriously, including all of the open source code repositories managed through our organizations.
```
Comment (ngo100ping) [inline .github/SECURITY.md]: Should be 
`open-source`

File: .github/SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Even though open source repositories are outside of the scope of our bug bounty program and therefore not eligible for bounty rewards, we will ensure that your finding gets passed along to the appropriate maintainers for remediation.
```
Comment (ngo100ping) [inline .github/SECURITY.md]: Should be 
`open-source`

File: .github/SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+| Version          | Supported          |
+| ---------------- | ------------------ |
+| develop          | :x:                |
+| staging          | :x:                |
+| master           | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability in any of our repository, please report it to us through coordinated disclosure.
```
Comment (ngo100ping) [inline .github/SECURITY.md]: Should be
`repositories`

File: .github/SUPPORT.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+We’re excited that you’re using **Flutter Boilerplate** and we’d love to help.
+To help us help you, please read through the following guidelines.
+
+Please understand that people involved with this project often do so for fun,
+next to their day job; you are not entitled to free customer service.
+
+## Contents
+
+*   [Questions](#questions)
+    *   [Asking quality questions](#asking-quality-questions)
```
Comment (ngo100ping) [inline .github/SUPPORT.md]: Fix indentation

File: .github/SUPPORT.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+Help us help you!
+
+Spending time framing a question and adding support links or resources makes it
+much easier for us to help.
+It’s easy to fall into the trap of asking something too specific when you’re
+close to a problem.
+Then, those trying to help you out have to spend a lot of time asking additional
+questions to understand what you are hoping to achieve.
+
+Spending the extra time up front can help save everyone time in the long run.
```
Comment (ngo100ping) [inline .github/SUPPORT.md]: Should be
`upfront`

File: .github/SUPPORT.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+*   Try to define what you need help with:
```
Comment (ngo100ping) [inline .github/SUPPORT.md:38]: Fix indentation

File: .github/SUPPORT.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    *   Is there something in particular you want to do?
```
Comment (ngo100ping) [inline .github/SUPPORT.md]: Should be
* Is there something, in particular, you want to do?

**Review by ngo100ping — CHANGES_REQUESTED** [review]
Please do the requested changes

File: .github/PULL_REQUEST_TEMPLATE/PRIMARY.md (hunk 1 above)
Comment (xscotophilic) [reply .github/PULL_REQUEST_TEMPLATE/PRIMARY.md]: fixed

File: .github/PULL_REQUEST_TEMPLATE/PRIMARY.md (hunk 2 above)
Comment (xscotophilic) [reply .github/PULL_REQUEST_TEMPLATE/PRIMARY.md]: fixed

File: .github/SECURITY.md (hunk 3 above)
Comment (xscotophilic) [reply .github/SECURITY.md]: fixed

File: .github/SECURITY.md (hunk 4 above)
Comment (xscotophilic) [reply .github/SECURITY.md]: fixed

File: .github/SECURITY.md (hunk 5 above)
Comment (xscotophilic) [reply .github/SECURITY.md]: fixed

File: .github/SUPPORT.md (hunk 6 above)
Comment (xscotophilic) [reply .github/SUPPORT.md]: fixed

File: .github/SUPPORT.md (hunk 7 above)
Comment (xscotophilic) [reply .github/SUPPORT.md]: fixed

File: .github/SUPPORT.md (hunk 8 above)
Comment (xscotophilic) [reply .github/SUPPORT.md:38]: fixed

File: .github/SUPPORT.md (hunk 9 above)
Comment (xscotophilic) [reply .github/SUPPORT.md]: fixed

**Review by ngo100ping — APPROVED** [review]

**Pull request merged**